### PR TITLE
[commons-lang] LPD-19869 Add petra-time and Remove org.apache.commons.lang.time.StopWatch usages

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:report:2552018191

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
found that I missed a usage in `UpgradeRecorderTest`
retesting

**Comment by tinatian** [comment]
ci:report:2552884896

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:report:2553371664

**Comment by julschong** [comment]
no unique failure
I think `LocalFile.Locale#CanUpdateGroupSettingsWithDifferentAvailableLanguages` is unrelated

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/search/test/UserReindexerPerformanceOfLargeUserGroupInManySitesTest.java [hunk 1]
```diff
@@ -280,11 +281,11 @@ protected void measure(
 
 		StopWatch stopWatch = new StopWatch();
 
-		stopWatch.start();
```
Comment (tinatian) [inline modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/search/test/UserReindexerPerformanceOfLargeUserGroupInManySitesTest.java:283]: @julschong This is wrong, you should not remove `stopWatch.start()`

**Comment by tinatian** [comment]
@julschong Please see my comment.
Also, please do:
1. Remove StopWatch#start(), as you can see all the start() are invoked after a "new StopWatch"
2. Remove StopWatch#stop(), almost all invocations to stop() are followed by a "getTime()".

Please resend.
Thanks.
Tina

**Pull request closed without merging**